package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "meanjs",
   "description": "Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js.",
-  "version": "0.4.2",
-  "meanjs-version": "0.4.2",
+  "version": "0.5.0-beta",
```

Comment:
I wanted to leave it generic on purpose. To me, RC1 indicates a specific release, where beta just says it's currently under development and changing.